anime-relations.txt

```diff
@@ -35,7 +35,7 @@
 - version: 1.3.0
 
 # Update this date when you add, remove or modify a rule.
-- last_modified: 2023-10-13
+- last_modified: 2024-06-01
```

Comment:
For future reference, this must be in `YYYY-MM-DD` format (e.g. `2024-01-06`). Thanks!

Reply:
Yeah no worries, the later ones are all correct, somehow got confused on this one xD.